themes-default/slim/views/history.mako

```diff
@@ -105,22 +109,22 @@ const startVue = () => {
             <div class="show-option">
                 <span>Limit:</span>
                     <select name="history_limit" id="history_limit" class="form-control form-control-inline input-sm">
-                        <option value="10" ${'selected="selected"' if limit == 10 else ''}>10</option>
-                        <option value="25" ${'selected="selected"' if limit == 25 else ''}>25</option>
-                        <option value="50" ${'selected="selected"' if limit == 50 else ''}>50</option>
-                        <option value="100" ${'selected="selected"' if limit == 100 else ''}>100</option>
-                        <option value="250" ${'selected="selected"' if limit == 250 else ''}>250</option>
-                        <option value="500" ${'selected="selected"' if limit == 500 else ''}>500</option>
-                        <option value="750" ${'selected="selected"' if limit == 750 else ''}>750</option>
-                        <option value="1000" ${'selected="selected"' if limit == 1000 else ''}>1000</option>
-                        <option value="0"   ${'selected="selected"' if limit == 0   else ''}>All</option>
+                        <option value="10" :selected="limit === 10">10</option>
```

Comment:
Cant this be a little cleaner using v-for?

Reply:
I forgot about `v-model`. 🙄